### ISSUE TITLE
[Bug]: RuntimeError: lsass.exe not found!

### ISSUE THREAD

**Issue by JoeanAmier** [thewh1teagle/rookie#72]
### What happened?

Failed to read cookie.

### Steps to reproduce

```
import rookiepy

print(rookiepy.chrome())
```

### What browsers are you seeing the problem on?

Chrome, Edge

### Relevant log output

```shell
Traceback (most recent call last):
  File "C:\Users\youyq\AppData\Roaming\JetBrains\PyCharm2024.2\scratches\scratch_2.py", line 3, in <module>
    print(rookiepy.chrome())
          ^^^^^^^^^^^^^^^^^
RuntimeError: lsass.exe not found!

Location:
    rookie-rs\src\windows\appbound\impersonate.rs:91:3
```

**Comment by thewh1teagle** [comment]
Chrck if lsass.exe exists in task manager and try to reboot.
What's the os version?

**Comment by thewh1teagle** [comment]
Try this version: 

```console
pip install rookiepy==0.5.6b0
```

**Comment by JoeanAmier** [comment]
Using version 0.5.6b0 can successfully read.

**Comment by thewh1teagle** [comment]
Fixed in v0.5.6
